package.json

```diff
@@ -461,13 +462,17 @@
     "os-name": "^3.1.0",
     "proxyquire": "^2.1.3",
     "remark-gfm": "^1.0.0",
+    "superagent": "*",
+    "tar-fs": "*",
     "toml": "^3.0.0",
     "uuid": "^8.3.2",
     "vscode-languageclient": "^6.1.3",
     "vscode-languageserver": "^6.1.1",
-    "vscode-languageserver-textdocument": "^1.0.1"
+    "vscode-languageserver-textdocument": "^1.0.1",
+    "zlib": "*"
   },
   "extensionDependencies": [
-    "ms-dotnettools.vscode-dotnet-runtime"
+    "ms-dotnettools.vscode-dotnet-runtime",
+    "yaozheng.vscode-pde"
```

Comment:
Is this extension used in the extracting script?

Reply:
its a pre-requisite to run the jdt server

Reply:
Ahh gotcha! When we added the ms-dotnettools.vscode-dotnet-runtime we had to go through a security review, should we wait for that before committing this to master? Releasing this as is means that all users who get the update would also automatically download this extension which sounds like is not totally necessary right now.

Reply:
let me remove this for this PR then. i can add it back later when we pass review

Reply:
We can still keep this in the PR and just not diff this to against the master branch -- I think it's still useful for progress on the java server